client/verta/tests/test_permissions/test_sharing_old.py

```diff
@@ -10,163 +10,67 @@
 
```

Comment:
Does the [`ORG_SCOPED_PUBLIC` concept](https://github.com/VertaAI/protos-all/blob/4f8323557c43ad324456bb8ee044a6169a87933b/protos/protos/public/common/CommonService.proto#L162-L173) still exist in our platform? 🤔 if not, we can probably just remove this whole file.

Reply:
no, in v2 we can't set individual resources permissions. Removed the test